eng/tools/versioning/set-dev.js

```diff
@@ -0,0 +1,43 @@
+let argv = require("yargs")
+  .options({
+    "build-id": {
+      type: "string",
+      describe: "build ID suffix to give the package (e.g. usually YYYYMMDD.r)",
+      demandOption: true
+    },
+    "repo-root": {
+      type: "string",
+      describe: "root of the repository (e.g. ../../../)",
+      demandOption: true
+    }
+  })
+  .help().argv;
+
+const path = require("path");
+const versionUtils = require("./VersionUtils");
+
+async function main(argv) {
+  const buildId = argv["build-id"];
+  const repoRoot = argv["repo-root"];
+
+  const rushSpec = await versionUtils.getRushSpec(repoRoot);
+
+  console.log(`Updating packages with Build ID ${buildId}`);
+
+  const targetPackageFiles = rushSpec.projects.map(project =>
+    path.resolve(path.join(repoRoot, project.projectFolder, "package.json"))
+  );
+
+  for (const targetFile of targetPackageFiles) {
+    const targetFileContents = await versionUtils.readFileJson(targetFile);
```

Comment:
See my other comment https://github.com/Azure/azure-sdk-for-js/pull/5135/files#r335648354 about updating how we plan to add the dev version.